### ISSUE TITLE
Bug with JustEnoughResources

### ISSUE THREAD

**Issue by kyazuki** [sp614x/optifine#3047]
System often says "OpenGL Error: 1283(Stack overflow)" on chat.
This may cause the bug(https://github.com/way2muchnoise/JustEnoughResources/issues/204).

**Comment by Studio528-real** [comment]
I don't think it really means anything, does anything abnormal happen besides the message? if not then it doesn't mean anything and go to "Video Setting -> Other -> OpenGL Errors -> Click the On to Off"  

If something abnormal happens when you open it up please say what it is.

**Comment by kyazuki** [comment]
JER GUI(Mob Drops & Villager Trades) show nothing with Optifine.
When I open the GUI, System says the error.

**Comment by Studio528-real** [comment]
@sp614x you know what could cause this? since this is the first instance of GL errors meaning something and I would have no idea, I've only had to tell people to turn them off since they get confused with the 1281 or 1280 ones which don't mean anything.
